doc/cephadm/services/osd.rst

```diff
@@ -1,7 +1,7 @@
 ***********
 OSD Service
 ***********
-.. _device management: ../rados/operations/devices
+.. _device management: https://docs.ceph.com/en/quincy/rados/operations/devices
```

Comment:
Hey @shrutiparvekar congrats on your first PR! It seems like some of our doc writers (Zac and Anthony) have some feedback. I'm taking a look on my end..

Reply:
@ljflores: Sorry for dumping that wall of text on you guys. I've thought about it, and finding these dead links is useful. If this is a useful exercise for Grace Hopper day, keep doing exactly this and I will take care of all the tedious business of situating @shrutiparvekar's discoveries in the Procrustean bed of our CI workflow.